game/gamestate.go

```diff
@@ -3,15 +3,29 @@ package game
 import (
 	"fmt"
 	"math/rand"
+	"risk/communication"
 	"sort"
 )
 
+type Phase int
+
+const (
+	ReinforcementPhase Phase = iota
+	AttackPhase
+	ManeuverPhase
+	EndPhase
+)
+
 // GameState represents the dynamic state of the game at any point. stuff that will change during the game, (everything except the map - which is static), and at some point even the rules.
 type GameState struct {

```

Comment:
please rename to `State` since it's already in package `game` to be consistent with Go's naming conventions (externally it's used as `game.State`)